module/input/SensorFilter/src/SensorFilter.hpp

```diff
@@ -279,8 +158,44 @@ namespace module::input {
 
             /// @brief Mahony filter integral gain
             double Kp = 0.0;
+
+            /// @brief Bool to determine whether to use ground truth from the simulator
+            bool use_ground_truth = false;
         } cfg;
 
+        /// @brief Number of actuatable joints in the NUgus robot
+        static const int n_servos = 20;
+
+        /// @brief tinyrobotics model of NUgus used for kinematics
+        tinyrobotics::Model<double, n_servos> nugus_model;
+
+        /// @brief Current support phase of the robot
+        WalkState::SupportPhase current_support_phase = WalkState::SupportPhase::LEFT;
+
+        /// @brief Transform from anchor {a} to world {w} space
+        Eigen::Isometry3d Hwa = Eigen::Isometry3d::Identity();
+
+        /// @brief Transform from torso {t} to anchor {a} space
+        Eigen::Isometry3d Hat = Eigen::Isometry3d::Identity();
```

Comment:
This also shouldn't be a member variable, it's set in the function and then using shortly afterwards with no other uses.